docs/apache-airflow-providers-trino/operators/trino.rst

```diff
@@ -0,0 +1,56 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/operator:TrinoOperator:
+
+TrinoOperator
+=============
+
+Use the :class:`TrinoOperator <airflow.providers.trino.operators.trino>` to execute
+SQL commands in a `Trino <https://trino.io/>`__ query engine.
+
+
+Using the Operator
+^^^^^^^^^^^^^^^^^^
+
+Use the ``trino_conn_id`` argument to connect to your Trino instance where
+the connection metadata is structured as follows:
+
+.. list-table:: Trino Airflow Connection Metadata
+   :widths: 25 25
+   :header-rows: 1
+
+   * - Parameter
+     - Input
+   * - Host: string
+     - Trino server host name (or) ip address
+   * - Login: string
+     - Trino user name
+   * - Port: string
+     - Set Port to connect to the Trino server, for example on AWS EMR, Trino uses port 8889
+
+An example usage of the TrinoOperator is as follows:
+
+.. exampleinclude:: /../../tests/system/providers/trino/example_trino.py
+    :language: python
+    :start-after: [START howto_operator_trino]
+    :end-before: [END howto_operator_trino]
+
+.. note::
+
+  This Operator can be used to run any syntactically correct Trino query(one at a time), but we cannot run
+  multiple queries in the same task.
```

Comment:
If we can lets point to Trino documentation that states this. This is a limitation coming from Trino not from Airflow so it's important that users will understand this.